pkg/tests/end_to_end_tests/drop_test.go

```diff
@@ -25,7 +26,7 @@ func TestSQLRetentionPeriod(t *testing.T) {
 		ts := []prompb.TimeSeries{
 			{
 				Labels: []prompb.Label{
-					{Name: MetricNameLabelName, Value: "TEST"},
+					{Name: utils.MetricNameLabelName, Value: "test"},
```

Comment:
is the "TEST" reduced to lower case on purpose?

Reply:
I guess, it happened while resolving conflicts.